core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1503,8 +1483,6 @@ secret_configs:
     build02_github_client_id:
       bw_item: build_farm_02_cluster
       field: github_client_id
-    sa.ci-operator.vsphere.config:
-      attachment: sa.ci-operator.vsphere.config
```

Comment:
@hongkailiu I'm not sure about the other removals, but this doesn't look right - it yields a yamllint error as there are two `bw_item` keys in this section now.

Reply:
Thanks.
Corrected.

Reply:
- The 2 removals of vsphere item: they are not needed in the secret (only vsphere sa, no others).
- Using cluster group instead of to each cluster: for syntax simplicity.

Joining a cluster into build farm has become lots of manual work now as we have more and more clusters in the farm and more and more places to modify.
I need think about the automation for this to avoid errors.